src/main/java/logisticspipes/utils/item/ItemStackRenderer.java

```diff
@@ -260,39 +256,6 @@ public void renderInGui() {
         GL11.glPopAttrib();
     }
 
-    public void renderInWorld() {
-        assert renderManager != null;
-        assert renderItem != null;
-
-        ItemIdentifier itemId = itemIdentifierStack.getItem();
-        EntityItem entityItem = entityCache.get(itemId);
-        if (entityItem == null) {
-            entityItem = new EntityItem(worldObj, 0, 0, 0, itemId.makeNormalStack(1));
-            entityItem.hoverStart = 0;
-            entityCache.put(itemId, entityItem);
-        }
```

Comment:
Is `entityCache` field still required and used in this class?

Reply:
Nice catch! I was planning to remove some other unused things from `ItemStackRenderer` but forgot that removing `entityCache` should be part of this PR.